lib/inputstreamhelper.py

```diff
@@ -206,7 +206,7 @@ def _arch(cls):
             if arm_version:
                 arch = 'armv' + arm_version.group()
         if arch in config.ARCH_MAP:
-            return config.ARCH_MAP[arch]
+            arch = config.ARCH_MAP[arch]
```

Comment:
In this case returning early would be fine.
So I would leave it as it was.

Reply:
This is a leftover from having "About" as settings page, where I didn't want to write `setSetting()` twice. Will revert.